test/custom-task-ctrls/wait-task-beta/cmd/controller/main.go

```diff
@@ -46,9 +46,34 @@ func newController(ctx context.Context, cmw configmap.Watcher) *controller.Impl
 		}
 	})
 
+	apiVersion := "wait.testing.tekton.dev/v1beta1"
+	kind := "Wait"
+
 	customruninformer.Get(ctx).Informer().AddEventHandler(cache.FilteringResourceEventHandler{
-		FilterFunc: tkncontroller.FilterRunRef("wait.testing.tekton.dev/v1beta1", "Wait"),
-		Handler:    controller.HandleAll(impl.Enqueue),
+		// TODO: Replace with the following once tkncontroller.FilterCustomRunRef is merged (fromhttps://github.com/tektoncd/pipeline/pull/5822)
```

Comment:
The relevant changes have been merged (https://github.com/tektoncd/pipeline/pull/5822) but unless we change the `wait-task-beta`'s `go.mod` to depend on a specific sha of `tektoncd/pipeline` instead of a release, we still need to wait for the next to release to switch this properly.